src/Raven.Studio/wwwroot/Content/ace/mode-rql.js

```diff
@@ -1037,3 +1037,7 @@ oop.inherits(Mode, TextMode);
 exports.Mode = Mode;
 
 });
+
+this.window.evalUnstrict = function (str) {
```

Comment:
Can we put this in some *.ts file?

Reply:
This was removed. I had to put it there to be outside the 'use strict' mode, which isn't possible in .ts file.